examples/with-xata/README.md

```diff
@@ -0,0 +1,55 @@
+# Next.js + Xata
+
+This example showcases how to use Next.js with [Xata](https://xata.io) as your data layer.
```

Comment:
The README still needs the follow the same format as other examples please.

Reply:
Can merge after that!

Reply:
Addressed!